core/templates/dev/head/pages/exploration-editor-page/improvements-tab/improvements-tab.directive.html

```diff
@@ -7,7 +7,7 @@ <h1 class="stat-value-with-rating"><[getOpenCardCount()]></h1>
       </div>
     </md-card>
   </div>
-  <md-card ng-repeat="card in getCards()" ng-if="isCardOpen(card)"
+  <md-card ng-repeat="card in getCards() | orderBy:isCardOpen:true"
```

Comment:
Is the resulting order stable? (i.e. when there are multiple open and multiple non-open threads, will you always get the same order when you reload?) If not it might be disconcerting and you might want to additionally sort by lastUpdated or similar.

Reply:
Good point, for now I've chosen title since it'll take quite a bit of refactoring to add lastUpdated (I don't think Playthroughs even have such a concept?)

I've filed #7336 as a starter project for someone to come along to add it, though!